client/src/components/rooms/transcriptionNotifications/CustomNotifications.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ActiveNotification, NotificationStack, usePropsFor } from '@azure/communication-react';
+import { Stack } from '@fluentui/react';
+import React, { useEffect } from 'react';
+
+export interface CustomNotificationsProps {
+  customNotifications?: ActiveNotification[];
+}
+
+export const CustomNotifications = (props: CustomNotificationsProps): JSX.Element => {
+  const { customNotifications } = props;
+  const notificationProps = usePropsFor(NotificationStack);
+  const [activeNotiifications, setActiveNotifications] = React.useState<ActiveNotification[]>([]);
+  useEffect(() => {
+    if (customNotifications) {
+      setActiveNotifications(notificationProps.activeErrorMessages.concat(customNotifications));
```

Comment:
Does resetting this mess with the dismissal logic? How does the customNotificaiton dismissal get tracked, wondering if new `notificationProps.activeErrorMessages` comes in will they reset the customNotificaiton

Reply:
there was a bug here that I fixed later on. this is exactly right, by doing this, this way I had issues seeing the other notifications. (I am also concattenating to the wrong array here but I noticed in a later PR)